spec/services/course_search_service_spec.rb

```diff
@@ -44,6 +44,18 @@
         end
       end
 
+      context "by distance" do
+        let(:sort) { "distance" }
+        let(:filter) do
+          { latitude: 54.9713392, longitude: -1.6112336 }
+        end
+
+        it "orders in descending order" do
+          expect(findable_scope).to receive(:by_distance).with(origin: [54.9713392, -1.6112336]).and_return(expected_scope)
+          expect(subject).to eq(expected_scope)
+        end
+      end
+
```

Comment:
Worth also adding a section to the request spec for ordering?

https://github.com/DFE-Digital/teacher-training-api/blob/2890-add-openapi-spec-for-courses/spec/requests/api/v3/courses/index_spec.rb#L93

Reply:
yep, now added